manifests/server.pp

```diff
@@ -487,11 +487,9 @@
   if $implementation == 'master' {
     $pm_service   = !$passenger and $service_fallback
     $ps_service   = undef
-    $rack_service = $passenger
   } elsif $implementation == 'puppetserver' {
     $pm_service   = undef
     $ps_service   = true
-    $rack_service = false
```

Comment:
I'm not sure why this was false to start with? I don't see how `$implementation == 'puppetserver'` and using passenger are mutually exclusive.

I guess if Puppet and Apache aren't sharing the same certs then I guess this change could potentially restart, Apache when something changed in Puppet also.